raster_analysis/results_store.py

```diff
@@ -178,20 +178,32 @@ def _get_ttl():
         )
 
     @staticmethod
-    def _get_batch_items(table_name, keys):
-        results_response = dynamodb_client().batch_get_item(
-            RequestItems={table_name: {"Keys": keys}}
-        )
-        results = results_response["Responses"][table_name]
-        if len(results) == 0:
-            return results
+    def _get_batch_items(table_name, keys) -> List:
+
+        results = []
+        # batch_get_item has 100 items limit when sending request so chunking keys list
+        chunk = 0
+        while True:
+            start = chunk * 100
```

Comment:
Could you make this 100 a constant with an informative name?

Reply:
good call! thanks